### ISSUE TITLE
Spreadsheet Formulas do not respect culture specific decimal separator

### ISSUE THREAD

**Issue by jivanova** [telerik/kendo-ui-core#2928]
### Bug report
When a culture with comma as a decimal separator is set to the page, Spreadsheet's cells respect the new separator, but the formula input remains with dot separator. In addition, using comma separated numbers in a formula results in a formula error.

### Reproduction of the problem
1. Use the following [DOJO](https://dojo.telerik.com/OzAKIYOz/2)
2. Click on cell D3 - although the cell shows 12,39 its value in formula bar is with a dot 12.39
3. Click on cell E3 - formula says `=C3*D3*0.2`

4. Try to change 0.2 to 0,2 (as the culture should use comma).
5. Insert the following simple formula in a cell `=4,1+6,3`

### Current behavior
Formula error - #VALUE! is displayed

### Expected/desired behavior
Formula respects culture and it uses comma, instead of dot for decimal separator.

### Environment
* **Kendo UI version:** 2017.1.118
* **Browser:** all

**Comment by umutozel** [comment]
Not seems to be working with @progress/kendo-ui@2019.2.619

**Comment by umutozel** [comment]
Still not working (tested with 2019.2.619 using the same DOJO)

**Comment by Dimitar-Goshev** [comment]
@umutozel Thank you for your feedback. In R1 2019 we released a new property [useCultureDecimals](https://docs.telerik.com/kendo-ui/api/javascript/ui/spreadsheet/configuration/useculturedecimals) to allow Spreadsheet to work with comma decimal separators properly. The [DOJO](https://dojo.telerik.com/@Dimitar-Goshev/AfiZUdEM) from the issue works as expected, when useCultureDecimals: true is set and Kendo UI version is R1 2019 or greater.

I updated the issue description to include this information as well.
